### DISCUSSION TITLE
Q: do we/should we care about small grammar mistakes?

### DISCUSSION THREAD

**Discussion by akuankka128** [rust-tutorials/learn-opengl#8] [Q&A] [answered]
If I notice small grammar mistakes on the Rust tutorials page, like "there's *[plural]*," does it matter enough for me to create a pull request to correct it? I wouldn't want to feel nitpicky, but I still want to do something about it.

**Comment by Lokathor** [comment, marked as answer]
it's low-effort to accept grammar fixes, so it shouldn't be a problem
